libraries/user.rb

```diff
@@ -0,0 +1,70 @@
+class Chef
+  # Chef Resource for declaring a user and group for Elasticsearch
+  class Resource::ElasticsearchUser < Resource
+    include Poise
+
+    actions(:create, :remove)
```

Comment:
What does everybody think about getting rid of the `()` parentheses for these DSL-like notations? I guess it would look & feel cleaner?